rubix-spi/src/main/java/com/qubole/rubix/spi/ClusterManager.java

```diff
@@ -53,10 +46,11 @@
 {
   private static Log log = LogFactory.getLog(ClusterManager.class);
 
-  private int currentNodeIndex = -1;
+  private String currentNodeName = "";
   private String nodeHostname;
   private String nodeHostAddress;
   private final AtomicReference<LoadingCache<String, List<String>>> nodesCache = new AtomicReference<>();
+  protected final ConsistentHash<SimpleNode> consistentHashRing = HashRing.<SimpleNode>newBuilder().build();
```

Comment:
@stagraqubole : for testing, should we use the default 1000 partition rate, or we should create a config and set it at lower value for tests?

Reply:
why would we want to use different value for tests?